### DISCUSSION TITLE
can I learn manim from doc only or what are the sources to learn full manim Features ?

### DISCUSSION THREAD

**Discussion by ghost** [ManimCommunity/manim#3504] [Q&A]
can I learn manim from doc only or what are the sources to learn full manim Features
- what about DevTaoism couses, is it Deserve to pay for its courses ?

**Comment by MrDiver** [comment]
In general you can learn Manim with only the Documentation it has a lot of examples that can be helpful, we are currently trying to figure out new ways to get users started with the library.
But in general you can always look at the reference manual and click through the classes.
One thing you should keep in mind that a base knowledge of python is very helpful when learning Manim because it basically relies on your programming skills.

I don't think there are any good "courses" at the moment for Manim, we have some videos on [Youtube Behackl](https://www.youtube.com/watch?v=rUsUrbWb2D4&list=PLsMrDyoG1sZm6-jIUQCgN3BVyEVOZz3LQ&ab_channel=BenjaminHackl)

Hope that Helps!
